cmd/builder/README.md

```diff
@@ -14,7 +14,7 @@ dist:
   output_path: /tmp/dist
 exporters:
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/alibabacloudlogserviceexporter v0.69.0
-  - gomod: go.opentelemetry.io/collector/exporter/loggingexporter v0.69.1
+  - gomod: go.opentelemetry.io/collector/exporter/debugexporter v0.69.1
```

Comment:
We should probably bump the version here as well, otherwise the example won't work (there is no debug exporter at v0.69.0)

Reply:
Good point, i've updated it to show 0.86.0, though folks reading this may run into the problem that it wont work until the next release. I think that's ok, just something to keep in mind.